apps/server/src/solana-contract/queue/pending-write-handler.service.ts

```
@@ -0,0 +1,91 @@
+					payload.contractId,
+					payload.milestone_index,
+					payload.new_status,
+					payload.new_evidence,
+				)
+				break
+
+			case 'SET_BALANCE':
+				const { data: escrow } =
+					await this.escrowFirestoreService.getDocByContractId(
+						payload.contractId,
+					)
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)